sdk/storagemover/Azure.ResourceManager.StorageMover/CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Specify the release day that we can release the beta package after merged

Reply:
Done